src/index.js

```diff
@@ -0,0 +1,105 @@
+import './style.css';
+
```

Comment:
- [ ] There's no implemented event listener. Kindly ensure that there is an event listener on the checkbox so as to trigger an event whenever the checkbox is checked or unchecked.